ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/StorageTrieNodeDataRequest.java

```diff
@@ -46,8 +46,12 @@ protected void doPersist(final Updater updater) {
 
   @Override
   public Optional<Bytes> getExistingData(final WorldStateStorage worldStateStorage) {
-    return worldStateStorage.getAccountStorageTrieNode(
-        getAccountHash().orElse(Hash.EMPTY), getLocation().orElse(Hash.EMPTY), getHash());
+    return getAccountHash()
+        .flatMap(
+            accountHash ->
+                worldStateStorage
+                    .getAccountStateTrieNode(accountHash, getHash())
```

Comment:
an issue in this call getAccountStateTrieNode instead of getAccountStorageTrieNode

Reply:
Thanks for catching this copypasta!